jwst/ramp_fitting/tests/test_ramp_fit.py

```diff
@@ -99,7 +99,7 @@ def test_one_group_two_ints_fit_ols():
     model1.data[1, 0, 50, 50] = 12.0
 
     slopes, cube, optional, gls_dummy = ramp_fit(
-        model1, 1024 * 30000., True, rnoise, gain, 'OLS', 'optimal', 'none', test_dq_flags)
+        model1, 1024 * 30000., True, rnoise, gain, 'OLS', 'optimal', 'none', dqflags.pixel)
 
```

Comment:
The OLS and GLS optional results are different, so there are two return values for the optional results.  The `optional` variable is the one for OLS.  Since this test exclusively for OLS, there is nothing to capture, so use `gls_dummy` as a variable name, since a four tuple is returned from `ramp_fit`.